<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kajiiiro_rest_server</class>
 <widget class="QMainWindow" name="kajiiiro_rest_server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kajiiiro_rest_server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_controller">
       <attribute name="title">
        <string>controller</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_status">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>status</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_start_stop">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>381</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>DB setting</string>
        </property>
        <widget class="QPushButton" name="pushButton_load">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>120</y>
           <width>98</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>load</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>221</width>
           <height>136</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_host">
              <property name="text">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_host">
              <property name="text">
               <string>localhost</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_db_name">
              <property name="text">
               <string>db name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_db_name">
              <property name="text">
               <string>kajiiiro_rest_db</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_user">
              <property name="text">
               <string>user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_user">
              <property name="text">
               <string>kajiiiro</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_pass">
              <property name="text">
               <string>pass</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_pass">
              <property name="text">
               <string>kajiiiro_pass</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <zorder>lineEdit_host</zorder>
        <zorder>lineEdit_db_name</zorder>
        <zorder>lineEdit_user</zorder>
        <zorder>lineEdit_pass</zorder>
        <zorder>label_host</zorder>
        <zorder>label_db_name</zorder>
        <zorder>label_user</zorder>
        <zorder>label_pass</zorder>
        <zorder>lineEdit_host</zorder>
        <zorder>pushButton_load</zorder>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_setting_info">
       <attribute name="title">
        <string>setting info</string>
       </attribute>
       <widget class="QLabel" name="label_information">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>341</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>message</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_help">
       <attribute name="title">
        <string>help</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKajiiiro_rest_server">
    <property name="title">
     <string>kajiiiro_rest_server</string>
    </property>
   </widget>
   <addaction name="menuKajiiiro_rest_server"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
